LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2011-2016 Joost Cassee and others
+Copyright (C) 2011-2019 Joost Cassee and others
```

Comment:
Can you make this, "... and contributors"?

Reply:
Done